documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -167,15 +167,45 @@ Ensure that the recreated secrets have exactly the same name, labels, and fields
 .. Apply the `KafkaUser` resource configuration:
 +
 [source,shell]
-kubectl apply -f <user_configuration_file>
+kubectl apply -f <user_configuration_file> -n my-project
 
-. Deploy the Kafka cluster using the original configuration for the `Kafka` resource:
+. Deploy the Kafka cluster using the original configuration for the `Kafka` resource.
+If it's a KRaft cluster, add the annotation `strimzi.io/pause-reconciliation="true"` to the YAML file. 
 +
 [source,shell]
 ----
 kubectl apply -f <kafka_resource_configuration>.yaml -n my-project
 ----
 
+.. KRaft only: recover the `clusterId` value from one of the volumes by spinning up a temporary pod:
```

Comment:
Maybe you could say something like `If you don't know the cluster ID of the cluster (e.g. from logs or copies of the Kafka custom resource)`.